__tests__/app.test.js

```diff
@@ -78,3 +78,27 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  it("should return status 200 and an array of all articles sorted by 'created_at' property in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        articles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            title: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+          expect(article).not.toHaveProperty("body");
+        });
```

Comment:
Really good test, but what if the articles array is empty? How could we add an assertion to ensure that there is atleast 1 item in the array?

Reply:
Yeah I thought of this, but couldn't figure out what to do if i add this, because even if there are not articles inside ATM, the request is still successful and at some point there will be articles, but i will add this check there